platform/working-with-vsp/vsp-collaboration-cycle/vsp-collaboration-cycle.md

```diff
@@ -313,11 +313,21 @@ You bring:
 - Link to your [test plan](https://dsvavsp.testrail.io/index.php?/suites/view/2&group_by=cases:section_id&group_order=asc) in TestRail with [test results](https://dsvavsp.testrail.io/index.php?/runs/view/7&group_by=cases:section_id&group_order=asc) _for QA practice area review_ 
 - Link to your [Coverage for References](https://dsvavsp.testrail.io/index.php?/reports/view/12) and [Summary(Defects)](https://dsvavsp.testrail.io/index.php?/reports/view/14) reports in TestRail _for QA practice area review_
 - Review the [staging accessibility review processes](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/staging-review-processes.md)
-- Link to verification that accessibility checks are complete:
-  - (preferred) Link to your [TestRail VSP accessibility test plan template](https://dsvavsp.testrail.io/index.php?/suites/view/14&group_by=cases:section_id&group_order=asc).
-  
-OR
-  - Link to a doc in your product folder that indicates all required checks from the accessibility [staging review prep](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/staging-review-processes.md) doc are complete.
+- Verify low-level accessibility tests are complete. Choose one of these 2 options:
+  - **(preferred)** Use the [TestRail VSP low-level accessibility testing checklist](https://dsvavsp.testrail.io/index.php?/suites/view/14&group_by=cases:section_id&group_order=asc).
+  <br/>**OR**
```

Comment:
What is the benefit of creating a separate doc in the team's product folder, rather than having them certify the checklist in the request ticket? It's easier for the teams and the a11y specialist if the checklist is in the staging review request ticket, imho.

Reply:
@jenstrickland That's a good point. So instead of copy/pasting it into a new doc, we'd change that line to ask teams to check off the list in the staging review or full audit issue they open.

@gmaciolek What do you think about this? I could go with that, letting PMs fill out the checklist in issue, instead of having to create a new doc, and link to it.

Reply:
I'm fine with that! I don't even recall how we arrived at linking to a doc in the product folder. I think it's probably a carry-over from when we were asking people to create a Markdown file using a template? But asking teams to certify the checks are complete in the ticket seems like a much better solution to me.

Reply:
Thank you both. I'll make these changes after a few meetings.